src/frontend/ai-reviewer-admin/.env.template

```diff
@@ -1,2 +1,5 @@
-REACT_APP_AI_REVIEWER_API_URL=http://localhost:8090/
-PORT=3002
\ No newline at end of file
+PORT = 3002
+REACT_APP_KEYCLOAK_REALM=ai-reviewer
+REACT_APP_KEYCLOAK_CLIENT_ID=ai-reviewer-ui
+REACT_APP_KEYCLOAK_URL=http://localhost:8081/auth
+REACT_APP_AI_REVIEWER_API_URL=http://localhost:8090/
```

Comment:
I get this in the Chrome console when I launch the application using docker-compose:
**env-config.js:2 Uncaught SyntaxError: Invalid or unexpected token
asyncToGenerator.js:6 Uncaught (in promise) clientId missing**

If I open up the env-config.js in Chrome, I see this:
![image](https://user-images.githubusercontent.com/55215368/119734641-2269d180-be30-11eb-88a0-e2b8810b6caf.png)

The reason the PORT=3002 was at the bottom of the file was to fix this issue.  In this commit, the PORT was moved to the top of the list and so there is an extra carriage return after the 3002 and spaces before and after the = is confusing javascript.

Reply:
I committed a small change to fix this error.
Just moving the PORT line to the bottom seems to fix it.